example/cycle/test/test.js

```diff
@@ -7,31 +7,25 @@ import * as actions from '../../actions';
 
 import { fetchReposByUser, searchUsers } from '../';
 
-function assertSourcesSinks(sources, sinks, main, done) {
-  const Time = mockTimeSource();
+function assertSourcesSinks(sources, sinks, main, done, timeOpts = {}) {
```

Comment:
Wondering whether this function is generic enough to be added to @cycle/time. @Widdershin what do you think? The API for mocking sources with multiple nested functions (select().events()...) could be like like: 'select:events': () => <obj passed to .diagram>